### PR TITLE
Fixed loss of `@keyframes` when multiple `animation-name` values

### PR DESCRIPTION
**Pull request by pfrankov**

### CODE REVIEW

**Comment by pfrankov** [comment]
1.5 month passed. Any chance to merge?

**Comment by XhmikosR** [comment]
@pfrankov: please fetch and rebase.

/CC @giakki for quick review.

**Comment by pfrankov** [comment]
Done

**Comment by pfrankov** [comment]
@XhmikosR @giakki guys?

**Comment by pfrankov** [comment]
@XhmikosR @giakki meanwhile one more month passed...

File: src/lib.js [hunk 1]
```diff
@@ -86,8 +86,13 @@ function getUsedAnimations(rules, memo) {
                 })
                 /* Keep loopin' */
                 .forEach(function (property) {
-                    /* If declared as animation, it should be in the form 'name Xs etc..' */
-                    accumulator.push(property.value.split(' ')[0]);
+                    if (property.property === 'animation-name' && property.value.indexOf(',')) {
+                        /* Multiple animations, separated by comma */
+                        accumulator.push.apply(accumulator, property.value.replace(' ', '').split(','));
+                    }else {
```
Comment (XhmikosR) [inline src/lib.js]: Missing space

**Comment by XhmikosR** [comment]
@pfrankov: @giakki seems busy with real life and I don't see him reviewing this. So, be patient.

**Comment by pfrankov** [comment]
@giakki done!

**Comment by giakki** [comment]
Looks good!

**Pull request merged**